packages/docs/docs/guides/rendering.md

```diff
@@ -102,49 +207,79 @@ export async function POST(request: NextRequest) {
 
 For more detail, see:
 
-- [Tree streaming](#tree-streaming)
+- [Partial streaming](#partial-streaming)
 - [Append-only streaming](#append-only-streaming)
 
-## When you just want a string
+### From the Server into React
 
 :::note Architectures
 This applies to the following architectures:
 
-- [Headless AI (no UI)](./architecture.mdx#headless-ai)
-- Any other case where you just want a string.
+- [Server-Side AI.JSX + Client UI](./architecture.mdx#architecture-3-server-side-aijsx--client-ui)
 
 :::
 
-To render your component to a string:
+Use the `useAIStream` hook to fetch content from your server endpoint:
```

Comment:
These docs show how to fetch from the endpoint, but they don't actually talk about what the endpoint itself should be, which is really where the rendering is happening.

Reply:
resolving this for now and will address this in the next update